docs/changes.rst

```diff
@@ -26,6 +26,10 @@ Released: not yet
 
 **Enhancements:**
 
+* Group the general options into logical groups in the help output. This creates
+  groups of commands (server, server configuration , general) (see issue
+  # 672)
+>>>>>>> Fixes issue #672- Organize cmd line options by group
```

Comment:
Remove this remainder from the merge conflict.

Reply:
DONE